compiler/rustc_resolve/src/lib.rs

```diff
@@ -667,10 +667,22 @@ impl<'ra> Module<'ra> {
         }
     }
 
+    fn for_each_child<'tcx, R, F>(self, resolver: &R, mut f: F)
```

Comment:
```suggestion
    fn for_each_child<R: AsRef<Resolver<'ra, '_>>>(self, resolver: &R, mut f: impl FnMut(&R, Ident, Namespace, NameBinding<'ra>))
```
(Although `'tcx` might be necessary.)